modules/rosa-how-upgrades-work.adoc

```diff
@@ -0,0 +1,45 @@
+
+[id="rosa-upgrade-scheduled-time_{context}"]
+== Cluster upgrade scheduled time
+
+The time at which cluster upgrade is set to start is called upgrade `Scheduled Time`. This is when the preparation for the cluster upgrade begins with pre-upgrade health checks and additional compute capacity creation. The actual cluster upgrade as seen by running `oc get clusterversion` command in the ROSA CLI starts within one hour from `Scheduled Time`. You receive a notification when the cluster upgrade starts.
+
+[id="rosa-cluster-upgrade-overview_{context}"]
+== {product-title} upgrade overview
+
+The following are the high-level steps carried out during the cluster update:
+
+. Before the cluster upgrade begins, a cluster health check is performed by the Managed Upgrade Operator (MUO). If the MUO identifies an issue that requires corrective action, you will be notified proactively. Some examples of the cluster health checks that MUO performs include:
+** Identifying any Pod Disruption Budgets (PDBs) that can potentially block or delay the update of the nodes.
+** Ensuring cluster Operators are available and healthy.
+** Discovering any nodes that are already not in a `Ready` status.
+. A temporary worker node is created in the cluster to allow for the scheduling of drained pods during the update.
+. The cluster version is set to the target OpenShift version.
++
+[NOTE]
+====
+In certain situations, an upgrade path can become unavailable since the time the cluster update was requested. In such cases, the upgrade is automatically canceled and a notification is sent. You must pick another target version to request the upgrade.
+====
++
+. During the upgrade, the control plane components are updated to the new version.
+. Next, individual cluster Operators perform update tasks on their domain of the cluster.
+. Finally, the `Machine Config` Operator (MCO) begins updating the system configuration and operating system of every node. During this step, each node is rebooted after successfully draining the workloads running on the node.
+.. During the update of each node, workloads are drained honoring the PDBs. Workloads with PDBs that do not allow disruptions essentially block the draining of the node, increasing the elapsed time for the cluster update.
+.. During the update of every node in the cluster, the cluster update waits until the time specified by the node drain grace period to allow for safely draining the workloads. Upon reaching the node drain grace period, the node is force drained to allow for cluster upgrade to progress. You can only configure the node drain grace period at cluster level before initiating the upgrade and you cannot change it after the cluster upgrade begins.
```

Comment:
suggest emphasizing this "node drain grace period" terminology through bold letters if possible.

Reply:
As per the [RH SSG](https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements), we use bold text for all graphical user interface (GUI) element names, and not to emphasize terms. I can italicize the first instance of 'node drain grace period' to emphasize the first appearance of the new term.